frontend/src/views/omni/Modals/DownloadInstallationMedia.vue

```diff
@@ -60,18 +60,11 @@ included in the LICENSE file.
 
       <t-input v-model="kernelArguments"/>
 
-      <!-- TODO(image-factory): enable this after further testing and making sure that it works -->
```

Comment:
I don't know if Omni uses this already, but Image Factory exports a list of supported platforms/images and secure boot support as well: https://github.com/siderolabs/image-factory/blob/main/pkg/metadata/platforms.go

Reply:
We could move it back to Talos machinery as well

Reply:
Omni doesn't use it yet. I guess it'll require a bit of a research.